### ISSUE TITLE
File list view complains fid is ambiguous

### ISSUE THREAD

**Issue by jlfranklin** [silkscreencms/silkscreen-issues#18]
With the PostgreSQL driver, the file manager view doesn't show any files and an exception is thrown:

```
Exception in Administer files[file_admin]: SQLSTATE[42702]: Ambiguous column: 7 ERROR: column reference "fid" is ambiguous LINE 7: GROUP BY fid, file_managed_filename, file_managed_uri, file_... ^
```

**Comment by jlfranklin** [comment]
This happens because the base table's index field, `fid`, is aliased as `fid` instead as `file_managed_fid`.  When the table is joined with `file_usage` on `fid`, PostgreSQL sees two `fid` fields, and the `GROUP BY fid` becomes (kinda) ambiguous.  

Adding the table prefix to the `GROUP BY` for base fields fixes this issue.

**Comment by jlfranklin** [comment]
Fixed in ab2dd7c65, closing.
